apps/backend/src/users/dto/get-user-response.dto.ts

```diff
@@ -0,0 +1,54 @@
+import {
```

Comment:
❓ What is the purpose of this dto?

Reply:
The purpose of this dto is to define the response schema for the `getUser` endpoint, which is basically the user entity minus the `applications` field (since we have a separate endpoint to get the application for an applicant and the `getUser` endpoint shouldn't return a user's applications). I checked the code now tho and it looks like the `getUser` endpoint actually returns a user of type `User` and not `GetUserResponse` so we'll change our code to reflect this!